element_array_ephys/export/nwb/nwb.py

```diff
@@ -215,7 +219,8 @@ def create_units_table(
             ephys.ProbeInsertion
             * ephys.CuratedClustering.Unit
             * probe.ProbeType.Electrode
-            & unit
+            & dict((k, unit[k]) for k in unit.keys()  # excess keys caused errs
+                    if k not in ['spike_times', 'spike_sites', 'spike_depths'])
```

Comment:
```suggestion
            & dict((k, unit[k]) for k in unit.keys()  # excess keys caused errs
                    if k not in ['cluster_quality_label', 'spike_count', 'spike_times', 'spike_sites', 'spike_depths'])
```

Reply:
Or do we need to restrict by the primary key of `ephys.CuratedClustering.Unit`?  Please carefully check this suggestion.  I may be misinterpreting something.

Reply:
My recollection is that I excluded exactly the keys that were needed, but I can retest with this suggestion